versions.props

```diff
@@ -1,4 +1,4 @@
-com.diffplug.spotless:spotless-plugin-gradle = 6.0.0
+com.diffplug.spotless:spotless-plugin-gradle = 5.15.0
```

Comment:
Should we prevent auto bumps?

Reply:
Good flag, added the `# dependency-upgrader:OFF` guard clause!